html-forms/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Coding Form!</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email:</label>
+        <div></div>
+        <input type="email" id="email" name="email" required />
```

Comment:
Generally you wouldn't want to just use a `div` as a line break here, but if you wanted a new line you nest the `input` inside of the `div`. This will then still give you the new line from the block level `div` element while also giving you the ability to style the `input` in relation to the `div` (future styling lessons)
```
<div>
  <input type="email" id="email" name="email" required />
<div>
```